File: src/test/java/com/github/brane08/fx/takebreak/ApplicationTest.java
package com.github.brane08.fx.takebreak;

import javafx.application.Application;
import javafx.fxml.FXMLLoader;
import javafx.stage.Stage;
import org.junit.jupiter.api.Test;

public class ApplicationTest extends Application {

    @Test
    public void testFxmlShow() {
        Application.launch(ApplicationTest.class);
    }

    @Override
    public void start(Stage primaryStage) throws Exception {
        var parent = FXMLLoader.load(getClass().getResource("/views/main.fxml"));
    }
}

File: src/test/kotlin/brane08/fx/takebreak/ApplicationTest.kt
package brane08.fx.takebreak

import javafx.application.Application
import javafx.fxml.FXMLLoader
import javafx.scene.Parent
import javafx.stage.Stage
import kotlin.test.Test


class ApplicationTest : Application() {

    @Test
    fun testFxmlShow() {
        Application.launch(ApplicationTest::class.java)
    }

    override fun start(p0: Stage?) {
        val parent = FXMLLoader.load<Parent>(this.javaClass.getResource("/views/main.fxml"))
    }

}